CONTRIBUTION.md

```diff
@@ -18,14 +18,13 @@ Open your code editor of choice and edit your desired component/package in
 
 ## Rules and Lintings
 
```

Comment:
@raphaellueckl why did u remove this? there was no team meeting where we agreed to change our rules

Reply:
this sentence is important to stay for those people that expect a Bootstrap-like framework.

Reply:
Oh, I forgot about this. This is a statement that is not true. The materials we export are not a webcomponent. Neither is the polyfill. This sentence is imho just confusing and does not really help. But we can discuss in the team.